pywb/templates/search.html

```diff
@@ -23,164 +23,107 @@ <h4 class="display-4">
             </div>
             <div class="col-12 invalid-feedback">
-               {% trans %}'Please enter a URL{% endtrans %}
+                {% trans %}'Please enter a URL{% endtrans %}
             </div>
         </div>
         <div class="form-row mt-2">
             <div class="col-5">
                 <div class="custom-control custom-checkbox custom-control">
                     <input type="checkbox" class="custom-control-input" id="open-results-new-window">
-                    <label class="custom-control-label" for="open-results-new-window">{{ _('Open results in new window') }}</label>
+                    <label class="custom-control-label" for="open-results-new-window">{{ _('Open results in new window')
+                        }}</label>
                 </div>
             </div>
             <div class="col-7">
                 <button type="submit" class="btn btn-outline-primary float-right" role="button" aria-label="Search">
-                    {% trans %}Search{% endtrans %}
+                    {% trans %}Go{% endtrans %}
                 </button>
-                <button class="btn btn-outline-info float-right mr-3" type="button" role="button"
-                        data-toggle="collapse" data-target="#advancedOptions"
-                        aria-expanded="false" aria-controls="advancedOptions" aria-label="Advanced Search Options">
+                <button class="btn btn-outline-info float-right mr-3" type="button" role="button" data-toggle="collapse"
+                    data-target="#advancedOptions" aria-expanded="false" aria-controls="advancedOptions"
+                    aria-label="Advanced Search Options">
                     {{ _('Advanced Search Options') }}
                 </button>
             </div>
         </div>
         <div class="collapse mt-3" id="advancedOptions">
-            <div class="form-group form-row">
-                <label for="match-type-select" class="col-sm-2 col-form-label" aria-label="Match Type">
-                    {{ _('Match Type:') }}
-                </label>
-                <select id="match-type-select" class="form-control form-control col-sm-6">
-                    <option value=""></option>
-                    <option value="prefix">{% trans %}Prefix{% endtrans %}</option>
-                    <option value="host">{% trans %}Host{% endtrans %}</option>
-                    <option value="domain">{% trans %}Domain{% endtrans %}</option>
-                </select>
-            </div>
             <p style="cursor: help;">
-               <span data-toggle="tooltip" data-placement="right"
-                     title="Restricts the results to the given date/time range (inclusive)">
-                   {{ _('Date/Time Range') }}
+                <span data-toggle="tooltip" data-placement="right"
+                    title="Restricts the results to the given date/time range (inclusive)">
+                    {{ _('Date Range (YYYY/MM/DD)') }}
```

Comment:
I know that @peterchanws wrote that the format is YYYY/MM/DD in the ticket, but it is strangely YYYYMMDD

Reply:
Thanks, I'll update. As for the other comment, I didn't test searching just that the page rendered the changes as all this PR changes is the labels... maybe the ID changed because of that (hence the `getElementById` error) /shrug.

Reply:
@edsu it appears to be caused (at least) by removing the `Match Type` section in the advanced search options. So it seems we can't just remove them, I'll work on simply making them hidden.